preferences.cc

```diff
@@ -172,6 +172,7 @@ Preferences::Preferences( QWidget * parent, Config::Class & cfg_ ):
   ui.autoScrollToTargetArticle->setChecked( p.autoScrollToTargetArticle );
   ui.escKeyHidesMainWindow->setChecked( p.escKeyHidesMainWindow );
   ui.darkMode->setChecked(p.darkMode);
+  ui.darkReaderMode -> setChecked(p.darkReaderMode);
```

Comment:
there is already a darkmode option in the preference dialog

Reply:
UI dark and Article view's dark should be separated. We have a yellow background article view theme. It isn't necessarily inverted. Also, goldendict has been historically able to load any user's CSS.

So, even on windows, there should be two options, one for UI, another for articleview.